### PR TITLE
BookStore: Initialize project

### PR DESCRIPTION
**Pull request by kessio**
**### Hello 😺** 
This project is the initialization of React bookstore website:

- Use React Router.
- Use React components.
- Linter test passed
- Create a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books.

**### Happy Reviewing**

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export default function Books() {
+  return (
+
+    <div className="pt-5 container">
+      <h3>List of books</h3>
+      <button className="btn-dark btn-small rounded" type="button">Remove book</button>
+      <div className="d-flex pt-5">
+        <input type="text" placeholder="Add your book" />
+      </div>
+    </div>
```
Comment (EricMbouwe) [inline src/components/Books.js]: - [x] You should render a list of books using the `Book` component, which doesn't exist right now. Let's create it.
- [x] You should render a form using the `CreateNewBook` component, which doesn't exist now. Let's create it.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @kessio, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Some components ✔
- Linters ✔

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (peterrobert) [reply src/components/Books.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @kessio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### [OPTIONAL]

- [ ] It is always good to customize your readme file. This will make your project more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**